cli/inspector.rs

```diff
@@ -856,8 +856,30 @@ impl v8::inspector::ChannelImpl for InspectorSession {
     call_id: i32,
     message: v8::UniquePtr<v8::inspector::StringBuffer>,
   ) {
+    
     let raw_message = message.unwrap().string().to_string();
-    let message = serde_json::from_str(&raw_message).unwrap();
+    let message: serde_json::Value = match serde_json::from_str(&raw_message) {
+      Ok(v) => v,
+      Err(error) => match error.classify() {
+        serde_json::error::Category::Syntax => json!({
+          "id": call_id,
+          "result": {
+            "result": {
+              "type": "error"
```

Comment:
missing comma here?

Reply:
No this is JSON - can't put a comma here :]

Reply:
I have no idea how this works the comma isn't optional this is JSON 😅 

good catch